CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
-
+* Improving useability for `ilr` and `ilr_inv` for 2D compositions.  ([#1393](https://github.com/biocore/scikit-bio/issues/1393))
```

Comment:
Can you expand the description of the bug? From this description, I can't tell what input(s) caused the bug and what the buggy behavior was (was it a silent bug producing incorrect results, or did it error noisily?). It appears the `basis` parameter didn't work with 1-D arrays for `ilr` and `ilr_inv`, but I'm unsure about the `clr_inv` transformation to `basis` that you added to `ilr_inv`. If there's more than one issue with `basis`, please describe that here.

The markdown heading below this section doesn't render correctly, please add a blank line after this bullet point.

Please also indicate the full public import path for these functions, i.e. `skbio.stats.composition.ilr` and `skbio.stats.composition.ilr_inv`.